app/packages/core/src/components/Sidebar/ViewSelection/ViewDialog.tsx

```diff
@@ -288,15 +292,15 @@ export default function ViewDialog(props: Props) {
                   colorOption?.color === initialColor)
               }
               sx={{
-                background: theme.common.black,
+                background: "#FF6D04",
```

Comment:
Will this work for both dark and light themes?
I would recommend adding a new token to the theme palette.